examples_tests/tests.py

```diff
@@ -32,6 +34,24 @@
 config = {}
 
 
+_MVN_SETTINGS = '<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0\n' + \
+    'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"\n' + \
+    'xsi:schemaLocation="http://maven.apache.org/SETTINGS/' + \
+    '1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">\n' + \
+    '  <interactiveMode>false</interactiveMode>\n' + \
+    '  <proxies>\n' + \
+    '    <proxy>\n' + \
+    '      <id>proxy</id>\n' + \
+    '      <active>true</active>\n' + \
+    '      <protocol>http</protocol>\n' + \
+    '      <host>{}</host>\n' + \
+    '      <port>{}</port>\n' + \
+    '      <nonProxyHosts>.internal</nonProxyHosts>\n' + \
+    '    </proxy>\n' + \
+    '  </proxies>\n' + \
+    '</settings>\n'
```

Comment:
If you put this string inside parentheses, you don't need the \ nor the +, which I find really useful.
There's also the option of using three quotes and textwrap.dedent, which I don't like that much but it's common.